warehouse/integrations/bigquery/bigquery.go

```diff
@@ -859,7 +859,9 @@ func (bq *BigQuery) connect(ctx context.Context, cred BQCredentials) (*bigquery.
 }
 
 func (bq *BigQuery) dedupEnabled() bool {
-	return bq.config.isDedupEnabled || bq.config.isUsersTableDedupEnabled
+	return bq.config.isDedupEnabled ||
+		bq.config.isUsersTableDedupEnabled ||
+		bq.warehouse.GetBoolDestinationConfig("enableDedup")
```

Comment:
@lvrach does it make sense to keep the other configuration parameters? Should we keep the opt-in `enableDedup` only?

Reply:
Add a way to disable the flag for all destinations.

Reply:
The flag coming from config.yaml will be called `forceMerge` (default `false`). The flag coming from the backend config should be called `enableAppend`.